easybuild/easyconfigs/p/python-parasail/python-parasail-1.1.12-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = "PythonPackage"
+
+name = 'python-parasail'
+modulename = name[len('python-'):]
+version = '1.1.12'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.org/project/parasail/'
+description = """ This package contains Python bindings for parasail. """
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://pypi.python.org/packages/source/p/%s' % modulename]
+sources = [{'filename': SOURCE_TAR_GZ, 'download_filename': '%s-%%(version)s.tar.gz' % modulename}]
```

Comment:
I would just hardcode `parasail` in both these lines, and drop `modulename` all together?